setup.py

```diff
@@ -21,6 +21,10 @@
     license='MIT',
     entry_points={
         'console_scripts': []},
+    short_description=(
+        'Engine for combining computational biology models with the '
+        'Vivarium interface.'
+    ),
```

Comment:
@eagmon does this description look good to you? If so, we can make the GitHub repo description match for consistency

Reply:
How about this slightly modified description: `Engine for composing and simulating computational biology models with the Vivarium interface`. Yes to consistency with the repo description!

Reply:
Done in 0bdd01e. I've also updated the repo description